modules/rostests/kmtests/kmtest/kmtest.c

```diff
@@ -316,8 +317,11 @@ main(
     if (ArgCount >= 1)
         AppName = Arguments[0];
 
-    if (ArgCount <= 1)
+    if (ArgCount != 2)
     {
+        if (ArgCount > 2)
```

Comment:
Wine tests frequently accept additional arguments that can be parsed by the test. So far kmtests haven't needed this feature, but it can easily be added. Prohibiting it at this point does not seem useful.